src/harness/testcases/WINNF_FT_S_MCP_testcase.py

```diff
@@ -410,16 +516,21 @@ def executeSingleMCPIteration(self, test_type, iteration_content, sas_test_harne
     if 'escRecords' in protected_entity_records:
       for esc_record in protected_entity_records['escRecords']:
         self._sas_admin.InjectEscSensorDataRecord({'record': esc_record})
-
+     
     if 'ppaRecords' in protected_entity_records:
       for ppa_record in protected_entity_records['ppaRecords']:
         self._sas_admin.InjectZoneData({'record': ppa_record})
 
+    if 'palRecords' in protected_entity_records:
```

Comment:
This needs to happen before PPA injection.

Reply:
Changed as per comment